### ISSUE TITLE
Action List Item: This LI does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); an element with a role set to the value: list as a parent; or a ul element (without an ARIA-assigned role), ol element (without an ARIA-assigned role) or element with a role set to the value 'list’ with an aria-owns attribute set to the ID of the element in the same DOM.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3833]
Description: This LI does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); an element with a role set to the value: list as a parent; or a ul element (without an ARIA-assigned role), ol element (without an ARIA-assigned role) or element with a role set to the value 'list’ with an aria-owns attribute set to the ID of the element in the same DOM.

Code Snippet:
<li aria-selected="false" id="fdp-list-item-16" class="fd-list__item fd-list__item--action" tabindex="-1" fd-list-item="" role="listitem"><button title="Action 1" aria-label="Action 1" class="fd-list__title">Action 1</button></li>

Expected: Ensure that whenever page elements are used proper list markup is provided.

Screenshot: 
![image](https://user-images.githubusercontent.com/32538291/98774714-93f0ae00-2411-11eb-811d-fcd545187f99.png)

**Comment by sKudum** [comment]
Platform component are wrapped on top of basic html so that in angular application developers can use the extra features provided along with basic html. For readability purpose added updated role for wrapped components which in turn gives a ul>li message format.
